Tutorials/NetKetSymmetryInterface.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
<p>This makes it sound like <code>Lattice</code>  doesn't have <code>.automorphisms</code>. </p><p> </p><p>Also, you may want to mention how there can be more automorphisms than space group elements (IIRC, the 4x4 grid with PBC had additional symmetries due to being isomorphic with a 2x2x2x2 hypercube.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1048/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>